ui/app/api/scripts/[scriptId]/route.ts

```diff
@@ -0,0 +1,20 @@
+import { ScriptsType } from '@/app/dto/ScriptsDTO';
+import prisma from '@/app/utils/prisma';
+import { NextRequest, NextResponse } from 'next/server';
+
+export async function GET(
+  _: NextRequest,
+  context: { params: { scriptId: string } }
+) {
+  const existingScript = await prisma.scripts.findFirst({
```

Comment:
we're getting rid of prisma

Reply:
will update this as needed after #1917 lands